longhaul-test/pubsub-workflow-deploy.yml

```diff
@@ -42,11 +42,11 @@ spec:
         dapr.io/app-port: "3000"
         dapr.io/log-as-json: "true"
         prometheus.io/scrape: 'true'
-        prometheus.io/port: '9988'  
+        prometheus.io/port: '9988'
     spec:
       containers:
       - name: pubsub-workflow
-        image: daprtests.azurecr.io/pubsub-workflow:dev
+        image: halspang/pubsub-workflow:dev
```

Comment:
Do we really want to refer to your repo?

Reply:
It's a pretty great repo, but no, no we do not :)